src/main/webapp/js/components/sidebar.jsx

```diff
@@ -99,13 +102,13 @@ class SideBar extends Component {
     return (
       <ul className="nav nav-second-level">
         <li className="border-none">
-          <Link to="/visitSchedule">
+          <Link to={`/viewEntity/${VISIT_ENTITY}`}>
```

Comment:
I think that those links should be also be removed from router. Moreover placeholder components (e.g. visit-schedule) should be removed.

Reply:
there should be a page to reschedule visits, but it will be a little different than the visit edit page

Reply:
fixed